### ISSUE TITLE
PyMuPDF Pro cannot extract Chinese content from DOC and DOCX files

### ISSUE THREAD

**Issue by maxyou2090** [pymupdf/PyMuPDF#3976]
### Description of the bug

The module can only extract numeric or English content and does not support Chinese.

### How to reproduce the bug

Code Sample
```
import pymupdf.pro

pymupdf.pro.unlock()
doc = pymupdf.open("/Users/maxyou/Downloads/demo.docx")
for page in doc:
    print(page.get_text())
    break
```

Output
```
PyMuPDFPro: Restricted Mode. Please visit https://pymupdf.io/try-pro to request your trial key.
hello,,123456789
```

DOCX Content
```
hello,中文示例,123456789
```

DOCX File
[demo.docx](https://github.com/user-attachments/files/17469592/demo.docx)

### PyMuPDF version

1.24.11

### Operating system

MacOS

### Python version

3.12

**Comment by ummm288** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by julian-smith-artifex-com** [comment]
Thanks for the bug report. This was fixed in release 1.24.12.
